website/docs/consumewinrtextension.md

```diff
@@ -0,0 +1,28 @@
+Enables the WinRT extension, C++/CX, for the specified projects/files.
+
+```lua
+consumewinrtextension "value"
+```
+
+### Parameters ###
+
+`value` one of:
```

Comment:
```suggestion
`value` is one of:
```

Reply:
Good catch. :+1:

Reply:
I had literally one job in my review and I failed 😅